### PR TITLE
take kanban 1.7.8.3 into account

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add 1.7.8.3 kanban 
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no issue
| How to test?      | 1.7.8.3 kanban should behave like 1.7.8.1 and 8.0.0 kanbans
| Possible impacts? |

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -60,6 +60,19 @@ const config = {
             doneColumnId: 15345341,
           },
         },
+        {
```
Comment (atomiix) [inline config.js:63]: Maybe you can get rid of the 1.7.8.1 then?

Comment (Progi1984) [reply config.js:63]: I approve @atomiix

Comment (matthieu-rolland) [reply config.js]: done !

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @matthieu-rolland

**Pull request merged**